plugins/hls-code-range-plugin/src/Ide/Plugin/CodeRange.hs

```diff
@@ -179,9 +180,13 @@ createFoldingRange :: CodeRange -> Maybe FoldingRange
 createFoldingRange (CodeRange (Range (Position lineStart charStart) (Position lineEnd charEnd)) _ ck) = do
     -- Type conversion of codeRangeKind to FoldingRangeKind
     let frk = crkToFrk ck
-
     Just (FoldingRange lineStart (Just charStart) lineEnd (Just charEnd) (Just frk))
 
+-- | Removes first node from folding ranges
+removeFirstNode :: [FoldingRange] -> [FoldingRange]
+removeFirstNode []     = []
+removeFirstNode (_:xs) = xs
```

Comment:
If you are worried that `tail` is not a total function, you can use [`drop 1`](https://hackage.haskell.org/package/base-4.17.0.0/docs/Prelude.html#v:drop)

Reply:
Makes sense, changed it.